reconcile/utils/openshift_resource.py

```diff
@@ -452,6 +454,7 @@ def add_desired(self, cluster, namespace, resource_type, name, value):
             except KeyError:
                 return None
             if name in desired:
+                self.register_error()
```

Comment:
why not?
```suggestion
                self.register_error(cluster)
```

Reply:
this is to indicate that a whole cluster is having issues. add_desired is at the namespace level. does that make sense?

Reply:
Well, now that you say it, it emerges a pattern 😂 

Most likely this usage that you describe would be best served with different error methods, but it is out of the scope of this PR :)